app/main.py

```diff
@@ -18,5 +18,5 @@ async def ping(settings: Settings = Depends(get_settings)):
         "ping": "pong",
         "environment": settings.environment,
         "testing": settings.testing,
-        "database_stage_url": settings.database_url,
+        "database_url": settings.database_url,
         }
```

Comment:
config your IDE to add new line at the end of the file after saving

Reply:
In the next PR I'm gonna add pre-commit